test/e2e/apibinding/apibinding_test.go

```diff
@@ -59,11 +59,11 @@ func TestAPIBinding(t *testing.T) {
 	t.Cleanup(cancel)
 
 	orgClusterName := framework.NewOrganizationFixture(t, server)
-	serviceProvider1Workspace := framework.NewWorkspaceFixture(t, server, orgClusterName)
-	serviceProvider2Workspace := framework.NewWorkspaceFixture(t, server, orgClusterName)
-	consumer1Workspace := framework.NewWorkspaceFixture(t, server, orgClusterName)
-	consumer2Workspace := framework.NewWorkspaceFixture(t, server, orgClusterName)
-	consumer3Workspace := framework.NewWorkspaceFixture(t, server, orgClusterName)
+	serviceProvider1Workspace := framework.NewWorkspaceFixture(t, server, orgClusterName, framework.WithShardConstraints(tenancyv1alpha1.ShardConstraints{Name: "root"}))
+	serviceProvider2Workspace := framework.NewWorkspaceFixture(t, server, orgClusterName, framework.WithShardConstraints(tenancyv1alpha1.ShardConstraints{Name: "root"}))
+	consumer1Workspace := framework.NewWorkspaceFixture(t, server, orgClusterName, framework.WithShardConstraints(tenancyv1alpha1.ShardConstraints{Name: "root"}))
+	consumer2Workspace := framework.NewWorkspaceFixture(t, server, orgClusterName, framework.WithShardConstraints(tenancyv1alpha1.ShardConstraints{Name: "root"}))
+	consumer3Workspace := framework.NewWorkspaceFixture(t, server, orgClusterName, framework.WithShardConstraints(tenancyv1alpha1.ShardConstraints{Name: "root"}))
```

Comment:
these are wrong. The root access is only needed to get the shard list. The actual test is not using root access.

Reply:
oha, fixed